android/build.gradle

```diff
@@ -8,8 +8,8 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.gradle:gradle-enterprise-gradle-plugin:3.10.1"
```

Comment:
revert this file, we can get the latest versions by merging dev into the release branch